### PR TITLE
unix_socket_path in RedisSettings

### PR DESCRIPTION
**Pull request by lazToum**
replaces #301

### CODE REVIEW

**Comment by lazToum** [comment]
From https://github.com/samuelcolvin/arq/pull/301, had to rebase.

**Comment by samuelcolvin** [comment]
okay, in future, best if you rebase or merge without creating a new pull request.

**Review by samuelcolvin — COMMENTED** [review]
otherwise looking good.

File: requirements/testing.in [hunk 1]
```diff
@@ -8,3 +8,5 @@ pytest-mock>=3,<4
 pytest-sugar>=0.9,<1
 pytest-timeout>=2,<3
 pytz
+redislite
+redis[hiredis]==4.2.2
```
[the remark this answers is not in the document]
Comment (samuelcolvin) [reply requirements/testing.in:12]: you'll need to re-run `pip-compile --output-file=requirements/testing.txt requirements/testing.in` too.

[the remark this answers is not in the document]
Comment (lazToum) [reply requirements/testing.in]: Done, thanks

**Comment by samuelcolvin** [comment]
thanks so much.

**Pull request merged**